### DISCUSSION TITLE
Out on vacation - September 7th to October 18th 2022

### DISCUSSION THREAD

**Discussion by Ylianst** [Ylianst/MeshCentral#4460] [Announcements]
Just a reminder that I will be out on vacation September 7th to October 18th. I will be traveling to France, landing and Paris and will tour the south coast of France. I will be on GitHub/Reddit/Telegram from time to time but will not be building new features for that time. I did record a lot of YouTube videos that will continue to premiere each Wednesday morning. I try to be a few videos ahead in normal times, but I have been working much more to have plenty of videos ready. I will have my development environment with me in case where is something really nasty I need to patch right away. Bryan will keep working on the MeshAgent while I am out and thank you to everyone that is answering user questions.

This vacation is part of the sabbatical benefit from my company. Every 7 years you work there, you get 8 weeks of sabbatical. I am taking a half one (4 weeks) this time around with a bit more of normal vacations. I am been at my company over 23 years now, so it's my 3rd sabbatical. Once I get back, I don't have any other big travel, I will get back to work on MeshCentral.

One thing to note is that I get way to many requests to possibly handle, so I apologize for not keeping up. There are some requests for new features that take many days to build and release. At the rate the requests are coming in, it's not possible to keep up. However, I do enjoy working on this.

**Comment by NiceGuyIT** [comment]
Enjoy your vacation @Ylianst . Try to stay of the grid and enjoy it.

**Comment by si458** [comment]
enjoy ur holiday 🏨 
we will keep up with users support questions for you!

**Comment by renueman** [comment]
Hope you have fun on your vacation.
